### ISSUE TITLE
Cannot Activate AN 2.0 on Linux Blender 2.8

### ISSUE THREAD

**Issue by ewhac** [JacquesLucke/animation_nodes#1188]
I'm running Debian Sid ("unstable") with Python 3.7.4.  I have successfully cloned the AN repo, checked out tag v2.0, and built an `animation_nodes.zip` package.  Blender 2.8 appears to install the resulting package successully.  However, when I click on the checkbox to activate it, the following error appears:

```
Traceback (most recent call last):
  File "/home/ewhac/bin/blender-2.80-linux-glibc217-x86_64/2.80/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/__init__.py", line 155, in <module>
    modules = import_modules.importAllSubmodules(__path__[0], __package__)
  File "/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/import_modules.py", line 8, in importAllSubmodules
    module = importlib.import_module("." + name, packageName)
  File "/home/ewhac/bin/blender-2.80-linux-glibc217-x86_64/2.80/python/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/base_types/nodes/vectorized_node.py", line 6, in <module>
    from .. effects import AutoSelectVectorization, VectorizeCodeEffect
ImportError: cannot import name 'AutoSelectVectorization' from 'animation_nodes.base_types.effects' (/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/base_types/effects/__init__.py)
```

Note that I am not using the Blender 2.8 Debian package from unstable/experimental, but the pre-compiled binary from Blender.org (v2.80.75).  My Python-fu is not strong.  Suggestions welcome.

**Comment by OmarSquircleArt** [comment]
This is probably due to the fact that files from older versions of Animation Nodes still exist in the addon directory. Please try deleting Animation Nodes manually (delete the `animation_nodes` directory) and try installing again.

**Comment by ewhac** [comment]
I had previously verified that the `$HOME/.config/blender/2.80/scripts/addons` directory was completely empty before installing AN.  The error still occurred.

The weirdness thickens.  After freshly restarting Blender this morning and trying again with the same .zip file, I get this different error:

```
Traceback (most recent call last):
  File "/home/ewhac/bin/blender-2.80-linux-glibc217-x86_64/2.80/scripts/modules/addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/__init__.py", line 155, in <module>
    modules = import_modules.importAllSubmodules(__path__[0], __package__)
  File "/home/ewhac/.config/blender/2.80/scripts/addons/animation_nodes/import_modules.py", line 8, in importAllSubmodules
    module = importlib.import_module("." + name, packageName)
  File "/home/ewhac/bin/blender-2.80-linux-glibc217-x86_64/2.80/python/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "animation_nodes/graphics/c_utils.pyx", line 6, in init animation_nodes.graphics.c_utils
NameError: name 'glVertex3f' is not defined
```
This error persists even after installing the python3-opengl package.

**Comment by OmarSquircleArt** [comment]
The `v2.0` branch doesn't support Blender 2.8. The `blender2.8` branch is what you are looking for.

**Comment by ewhac** [comment]
> The `v2.0` branch doesn't support Blender 2.8. The `blender2.8` branch is what you are looking for.

I was hoping to build the stable AN 2.0 version.  But if that's what's available for Blender 2.8, then that's what's available.

Successfully built and installed; thank you.
